manifests/s/SKDataAS/ORMCore/202208262/SKDataAS.ORMCore.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=QUSU.5-1-22000-832
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SKDataAS.ORMCore
+PackageVersion: "202208262"
+PackageLocale: en-US
+Publisher: SK Data AS
+PublisherUrl: https://www.skdata.no
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Dennis Kanon
+PackageName: ORM COre
```

```suggestion
PackageName: ORM_Core
```
This must match Control Panel.